### PR TITLE
remove use of 'roslyn' from option names as they can refer to several…

### PR DESCRIPTION
**Pull request by jmarolf**
… non-roslyn github repositories now

### CODE REVIEW

File: src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.UpdateFiles.cs [hunk 1]
```diff
@@ -24,17 +24,17 @@ internal static string GetDevDivPackagesDirPath(BuildVersion version)
...
             }
 
             return Path.Combine(
-                Options.RoslynDropPath,
-                Options.RoslynBuildQueueName,
-                Path.GetFileName(Options.RoslynBranchName), // The folder under Roslyn-Signed is just the last component of the name
-                Options.RoslynBuildConfig,
+                Options.GithubBuildDropPath,
+                Options.GithubBuildQueueName,
+                Path.GetFileName(Options.GithubBranchName), // The folder under Roslyn-Signed is just the last component of the name
```
Comment (jasonmalinowski) [inline src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.UpdateFiles.cs]: Comment needs updating?

**Comment by jmarolf** [comment]
> Should we just call it BuildQueueName and BranchName?

Sounds like a good idea.

**Comment by jmarolf** [comment]
@jasonmalinowski ready for review

**Comment by jmarolf** [comment]
did a fixup to update the text of `RIT /?`

**Comment by jmarolf** [comment]
ping @jasonmalinowski

**Pull request merged**